packages/theme-check-common/src/types/theme-liquid-docs.ts

```diff
@@ -23,12 +23,18 @@ export interface ThemeDocset {
 /**
  * JSON schemas resources for themes.
  */
-export interface JsonSchemaValidators {
+export interface JsonValidationSet {
   /** Whether it was augmented prior to being passed. */
   isAugmented?: boolean;
 
   /** Retrieves the JSON schema validator for theme sections. */
   validateSectionSchema(): Promise<ValidateFunction>;
+
+  /** Retrieves the JSON schema of the {% schema %} JSON blobs in sections/*.liquid files */
+  sectionSchema(): Promise<string>;
```

Comment:
I've followed you on your train of thought and am happy with where you stopped :D 

sectionGroupSchema Makes sense since those are documented as [Section Groups](https://shopify.dev/docs/themes/architecture/section-groups) on Shopify.dev